docs/cloud-native-security/d4c-get-started.asciidoc

```diff
@@ -43,23 +43,28 @@ One of the <<d4c-default-policies, default D4C policies>> sends process telemetr
 
 In order to detect threats using this data, you'll need active <<detection-engine-overview, detection rules>>. Elastic has prebuilt detection rules designed for this data. (You can also create your own <<rules-ui-create, custom rules>>.)
 
-To install and enable Elastic's prebuilt rules:
+To install and enable the prebuilt rules:
 
 . Go to *Security > Manage > Rules*, then click *Add Elastic rules*.
-. Use the *Tags* selector to search for `container`. Select the `Data Source: Elastic Defend for Containers` tag. The rules table displays relevant rules.
-. Select all the displayed rules and click *Install _x_ selected rule(s)*.
-. Once the rules have loaded, you will see the Rules management page. Use the *Tags* selector to search for `container`. Select the `Container Workload Protection` tag.
+. Use the *Tags* selector to search for `container`. Select the `Data Source: Elastic Defend for Containers` tag.
```

Comment:
Looks like we [refer](https://www.elastic.co/guide/en/security/8.9/rules-ui-management.html#sort-filter-rules) to this the Tags filter. Also, instead of telling users to search for part of a tag, can we tell users to search for the full tag name (e.g., `Data Source: Elastic Defend for Containers`)?

```suggestion
. Click the *Tags* filter next to the search bar, and search for the `Data Source: Elastic Defend for Containers` tag.
```